app/src/main/java/io/intrepid/contest/base/BaseFragmentActivity.java

```diff
@@ -24,12 +24,16 @@ protected void onCreate(Bundle savedInstanceState) {
         if (savedInstanceState == null) {
             Intent intent = getIntent();
             Fragment fragment = createFragment(intent);
-            getSupportFragmentManager()
-                    .beginTransaction()
-                    .add(R.id.fragment_container, fragment)
-                    .commit();
+            hostFragment(fragment);
         }
     }
 
+    protected void hostFragment(Fragment fragment) {
```

Comment:
Doesnt look like we use this anywhere outside this baseFragmentActivity, lets not change its behavior to keep it consistent with the skeleton project.